platforms/documentation/docs/src/docs/userguide/dep-man/centralizing-dependencies/centralizing_catalog_platform.adoc

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[platforms-vs-catalog]]
+= Using Catalogs with Platforms
+
+Both platforms and version catalogs help manage dependency versions in a project, but they serve different purposes and have different effects on dependency resolution:
+
+== Version Catalogs
+
+- **Purpose:** A version catalog is used to centralize and standardize dependency coordinates (group, name, version). It provides type-safe accessors in the build script, making dependencies easier to manage.
+- **Effect on Dependency Graph:** Version catalogs **do not** affect dependency resolution. The versions defined in the catalog are **suggestions** and **transparent to downstream consumers**. These versions can be overridden by other mechanisms like transitive dependencies or platforms.
```

Comment:
❗ Can we not say "suggestions" here?

Nothing in a catalog is 1) applied to any configuration by default - the contents need to be explicitly referenced from a `dependencies` block - true, but when they are then 2) they work the same as any "locally" declared dependency.  I don't know how to explain this briefly with one word, but I think it's really important to make this very clear.

I'd also explain that transparent means that the source of a dependency declaration (local or catalog) cannot be identified by downstream consumers.  The declaration itself is not transparent.

Reply:
Done. Might need a final check.